proto/ibc/core/client/v1/event.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+package ibc.core.client.v1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types";
+
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+
+// EventCreateClient is a typed event emitted on creating client
+message EventCreateClient {
+    string client_id = 1;
+
+    string client_type = 2;
+
+    google.protobuf.Any consensus_height  = 3 [(cosmos_proto.accepts_interface) = "Height"];
```

Comment:
why is this an Any? We can just use the height proto definition. The interface is only created to avoid circular dependencies

Reply:
Actually, to use `exported.Height` in typed events, we converted it to Any. But to pack `exported.Height` to Any, `Height` should be a pointer. That's the reason, we made it pointer. But, just got realized, we can use height proto definition directly. Thanks @colin-axner , we will revert these changes and use height proto definition.